extensions/wikia/AchievementsII/services/AchAwardingService.class.php

```diff
@@ -124,22 +124,22 @@ public function processSharing( $articleID, $sharerID, $IP ) {
 		 *  ]
 		 * ]
 		 */
-		if (empty( $this->mCounters[BADGE_SHARING] ) ) {
-			$this->mCounters[BADGE_SHARING] = array( 'ips' => array(), 'article_ids' => array() );
+		if( empty( $this->mCounters[ BADGE_SHARING ] ) ) {
+			$this->mCounters[ BADGE_SHARING ] = array( 'ips' => array(), 'article_ids' => array() );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "article_ids" 7 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)